src/Feed.php

```diff
@@ -104,6 +104,9 @@ private static function fromAtom(SimpleXMLElement $xml)
 
 		// generate 'timestamp' tag
 		foreach ($xml->entry as $entry) {
+			$link = (string)$entry->link['href'];
+			unset($entry->link);
```

Comment:
this code is just
```php
`$entry->link = (string)$entry->link['href'];`
```
But

Without `unset($entry->link);` not possible to rewrite property `$entry->link`.  
I.e. `$entry->link = (string)$entry->link['href'];` have no effect